### PR TITLE
run black on all files

### LINKED ISSUE #84: run black on all files

### PR DESCRIPTION
**Pull request by IKostric**
Fixes #84

Runs Black on all files to make future PRs cleaner.

### CODE REVIEW

File: moviebot/core/shared/intents/agent_intents.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 constants. This approach will help the Dialogue Policy to decide the intents
 from a pre-defined set."""
 
-__author__ = 'Javeria Habib'
+__author__ = "Javeria Habib"
```
Comment (NoB0) [inline moviebot/core/shared/intents/agent_intents.py:5]: Nit: remove `__author__`

File: moviebot/core/shared/intents/user_intents.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 This approach will help the NLU to identify the intents from a limited set
 without making a mistake."""
 
-__author__ = 'Javeria Habib'
+__author__ = "Javeria Habib"
```
Comment (NoB0) [inline moviebot/core/shared/intents/user_intents.py:5]: Nit: remove `__author__`.

File: moviebot/agent/agent.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
 """Types of conversational agents are available here."""
 
-__author__ = 'Javeria Habib'
+__author__ = "Javeria Habib"
```
Comment (NoB0) [inline moviebot/agent/agent.py:3]: Nit: remove `__author__`.

File: moviebot/dialogue_manager/dialogue_act.py [hunk 4]
```diff
@@ -1,7 +1,7 @@
 """Dialogue Act defines the action user or agent takes during the conversation.
 """
 
-__author__ = 'Javeria Habib'
+__author__ = "Javeria Habib"
```
Comment (NoB0) [inline moviebot/dialogue_manager/dialogue_act.py:4]: Nit: remove `__author__`.

File: moviebot/nlu/annotation/item_constraint.py [hunk 5]
```diff
@@ -1,7 +1,7 @@
 """The Dialogue Act Item defines the parameters for a Dialogue Act.
 It comprises ot the parameters mentioned in the conversation"""
 
-__author__ = 'Javeria Habib'
+__author__ = "Javeria Habib"
```
Comment (NoB0) [inline moviebot/nlu/annotation/item_constraint.py:4]: Nit: remove `__author__`.

File: moviebot/nlu/annotation/operator.py [hunk 6]
```diff
@@ -2,7 +2,7 @@
 It will be used to identify dialogue act item operator
 """
 
-__author__ = 'Javeria Habib'
+__author__ = "Javeria Habib"
```
Comment (NoB0) [inline moviebot/nlu/annotation/operator.py:5]: Nit: remove `__author__`.

File: moviebot/nlu/annotation/slots.py [hunk 7]
```diff
@@ -1,7 +1,7 @@
 """This file contains the list of slots available in the database.
 This approach will help the NLU to correctly identify slots"""
 
-__author__ = 'Javeria Habib'
+__author__ = "Javeria Habib"
```
Comment (NoB0) [inline moviebot/nlu/annotation/slots.py:4]: Nit: remove `__author__`.

File: moviebot/nlu/annotation/values.py [hunk 8]
```diff
@@ -3,7 +3,7 @@
 This approach will help the dialogue state tracker to update state for specific
 type of slot values easily"""
 
-__author__ = 'Javeria Habib'
+__author__ = "Javeria Habib"
```
Comment (NoB0) [inline moviebot/nlu/annotation/values.py:6]: Nit: remove `__author__`.

File: moviebot/nlu/nlu.py [hunk 9]
```diff
@@ -2,7 +2,7 @@
 NLU understands the user requirements and intents for the system to generate an
 appropriate response."""
 
-__author__ = 'Javeria Habib'
+__author__ = "Javeria Habib"
```
Comment (NoB0) [inline moviebot/nlu/nlu.py:5]: Nit: remove `__author__`.

File: moviebot/ontology/ontology.py [hunk 10]
```diff
@@ -1,7 +1,7 @@
 """Ontology is a class that loads ontology files (in .json format) into
 IAI MovieBot."""
 
-__author__ = 'Javeria Habib'
+__author__ = "Javeria Habib"
```
Comment (NoB0) [inline moviebot/ontology/ontology.py:4]: Nit: remove `__author__`.

File: moviebot/recorder/recorder_bot.py [hunk 11]
```diff
@@ -2,7 +2,7 @@
 If required, it also saves the dialogue acts.
 """
 
-__author__ = 'Javeria Habib'
+__author__ = "Javeria Habib"
```
Comment (NoB0) [inline moviebot/recorder/recorder_bot.py:5]: Nit: remove `__author__`.

File: moviebot/nlu/annotation/rule_based_annotator.py [hunk 12]
```diff
@@ -1,7 +1,7 @@
 """This file contains a class which can be used to annotate slot values
 in the user utterance based on rules and keyword matching."""
 
-__author__ = 'Javeria Habib'
+__author__ = "Javeria Habib"
```
Comment (NoB0) [inline moviebot/nlu/annotation/rule_based_annotator.py:4]: Nit: remove `__author__`.

File: moviebot/nlu/annotation/rule_based_annotator.py [hunk 13]
```diff
@@ -59,22 +80,22 @@ def slot_annotation(self, slot, user_utterance):
...
+            utterance:
+            raw_utterance:
 
         """
         # utterance = utterance.replace('?','')
         if slot in [x.value for x in [Slots.ACTORS, Slots.DIRECTORS]]:
             params = self._person_name_annotator(user_utterance)
         else:
-            func = getattr(self, '_' + slot + '_annotator')
+            func = getattr(self, "_" + slot + "_annotator")
```
Comment (NoB0) [inline moviebot/nlu/annotation/rule_based_annotator.py:92]: Nit: use fstring.

**Review by NoB0 — APPROVED** [review]
LGTM. See inline comments.
Please create a follow-up issue to fix incomplete docstrings (might want to create an issue per package).

File: moviebot/agent/agent.py (hunk 3 above)
Comment (IKostric) [reply moviebot/agent/agent.py:3]: Done in another PR.

File: moviebot/core/shared/intents/agent_intents.py (hunk 1 above)
Comment (IKostric) [reply moviebot/core/shared/intents/agent_intents.py:5]: Done in another PR.

File: moviebot/dialogue_manager/dialogue_act.py (hunk 4 above)
Comment (IKostric) [reply moviebot/dialogue_manager/dialogue_act.py:4]: Done in another PR.

File: moviebot/dialogue_manager/dialogue_context.py [hunk 14]
```diff
@@ -3,7 +3,7 @@
 recommendations.
 """
 
-__author__ = 'Javeria Habib'
+__author__ = "Javeria Habib"
```
[the remark this answers is not in the document]
Comment (IKostric) [reply moviebot/dialogue_manager/dialogue_context.py:6]: Done in another PR.

File: moviebot/dialogue_manager/dialogue_state_tracker.py [hunk 15]
```diff
@@ -1,6 +1,6 @@
 """Dialogue State Tracker updates the current dialogue state."""
 
-__author__ = 'Javeria Habib'
+__author__ = "Javeria Habib"
```
[the remark this answers is not in the document]
Comment (IKostric) [reply moviebot/dialogue_manager/dialogue_state_tracker.py:3]: Done in another PR.

File: moviebot/nlu/annotation/item_constraint.py (hunk 5 above)
Comment (IKostric) [reply moviebot/nlu/annotation/item_constraint.py:4]: Done in another PR.

File: moviebot/nlu/annotation/operator.py (hunk 6 above)
Comment (IKostric) [reply moviebot/nlu/annotation/operator.py:5]: Done in another PR.

File: moviebot/nlu/annotation/rule_based_annotator.py (hunk 12 above)
Comment (IKostric) [reply moviebot/nlu/annotation/rule_based_annotator.py:4]: Done in another PR.

File: moviebot/nlu/annotation/slots.py (hunk 7 above)
Comment (IKostric) [reply moviebot/nlu/annotation/slots.py:4]: Done in another PR.

File: moviebot/nlu/annotation/values.py (hunk 8 above)
Comment (IKostric) [reply moviebot/nlu/annotation/values.py:6]: Done

File: moviebot/nlu/nlu.py (hunk 9 above)
Comment (IKostric) [reply moviebot/nlu/nlu.py:5]: Done

File: moviebot/ontology/ontology.py (hunk 10 above)
Comment (IKostric) [reply moviebot/ontology/ontology.py:4]: Done

File: moviebot/recorder/recorder_bot.py (hunk 11 above)
Comment (IKostric) [reply moviebot/recorder/recorder_bot.py:5]: Done

**Review by WerLaj — APPROVED** [review]
LGTM

**Pull request merged**